fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1250,4 +1250,16 @@
      */
     @ConfField(mutable = true, masterOnly = true)
     public static boolean enable_batch_delete_by_default = false;
+
+    /**
+     * Used to set default db data quota bytes.
+     */
+    @ConfField(mutable = true, masterOnly = false)
+    public static long default_db_data_quota_bytes = 1024 * 1024 * 1024 * 1024L; // 1TB
```

Comment:
Both config should be `masterOnly = true`

Reply:
Yes, that's right. I change "masterOnly" to true

Reply:
OK， I change the "masterOnly" property of default_db_data_quota_bytes to true